src/config.rs

```diff
@@ -70,7 +70,7 @@ mod default {
         12
     }
 
-    pub fn raft_data_dir() -> String {
+    pub fn wal_path() -> String {
         "raft-data-dir".into()
     }
```

Comment:
Why do we change the name of a config entry? To be consistent with controller's config?